### PR TITLE
adjusted split function in training service and added description for…

### PR DESCRIPTION
**Pull request by ncrimi**
Created an additional data split such that the data is splitted into training, validation and test sets.
Imported the sklearn library for split_training_test_data.

### CODE REVIEW

File: training-service/src/services/training.py [hunk 1]
```diff
@@ -44,16 +53,35 @@ def data_preprocessing(training_data):
...
+Fetches best model from db and converts the db file into SH model
+Accuracy is computed before (with validation set) & after (with test set) training the fetched model
+If accuracy of the fetched model is higher after training than the current used model the new model will be saved to 
+the current directory such that the old model will be overwritten
+"""
 def improve_model(X, T, lang_name, training_data):
-    X_train, T_train, X_val, T_val = split_data(X, T)
+    #RECALL: return value of split_data function: X_train, X_val, T_train, T_val
+    
+    #split the data in 80:10:10 for train:valid:test dataset
```
[the remark this answers is not in the document]
Comment (ncrimi) [reply training-service/src/services/training.py:67]: we compute the current accuracy with the validation set, train the model with the training set and then test the model with the test set. In this way, we ensure that for each step an independent data set is used. before we used the validation before and after the training process.
It's a detail but it seems to me smoother. I read the the following article before implementing this strategy: 
- https://towardsdatascience.com/how-to-split-data-into-three-sets-train-validation-and-test-and-why-e50d22d3e54c
- https://machinelearningmastery.com/difference-test-validation-datasets/
